app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt

```diff
@@ -76,6 +76,10 @@ class OrdersUnderUserFragment : Fragment(), BottomIconDoubleClick {
         ordersUnderUserVM.connection
             .nonNull()
             .observe(viewLifecycleOwner, Observer { isConnected ->
+                if (!isConnected)
+                    showNoInternetScreen(true)
+
+                else {
```

Comment:
Still wrong indentation. I don't know why the build didn't fail. It should have

Reply:
Do you want me to change that?

Reply:
That's why I commented

Reply:
Done sir.

Reply:
Not resolved

Reply:
reason?  I removed the blank line which you apparently told was a wrong indentation practice.

Reply:
Not really. Blank line is not related to indentation, I have commented on the indentation if else block

Reply:
sir, is it fine now?

Reply:
sir, is it fine now?